cache/src/main/java/com/example/expenselogger/cache/di/CacheModule.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.expenselogger.cache.di
+
+import android.content.Context
+import com.example.expenselogger.cache.ExpenseDatabase
+import com.example.expenselogger.cache.dao.ExpenseDao
+import com.example.expenselogger.cache.repository.ExpenseRepository
+import com.example.expenselogger.cache.repository.ExpenseRepositoryImpl
+import dagger.Binds
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@[Module InstallIn(SingletonComponent::class)]
+internal interface CacheModule {
+
+    @get:Binds
+    public val ExpenseRepositoryImpl.expenseRepository: ExpenseRepository
```

Comment:
Make this internal

Reply:
Okay, I'll change it. I was actually thinking about this, decided to make it public because I felt since it exposes the repository interface; it might be required outside of this module